actions/installer/action.yml

```diff
@@ -45,4 +52,5 @@ runs:
         REPOSITORY: "${{ github.repository }}"
       working-directory: actions/installer/dist
       shell: bash
-      run: nodejs index.js 
+      run: npm start
+      
```

Comment:
note: I don't think we need:
- repository option for the Action input. It's not used by the code, is it?
- composite action. ACTION_REF is accessible by TS actions, so we should be able to simplify action.yaml like in https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-an-action-metadata-file (calling dist/index.js)?

Reply:
Done